content/en/docs/ibm/using-icr.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Using IBM Cloud Container Registry (ICR)"
+description = "Instructions for using private images from ICR"
+weight = 15
++++
+
+This guide describe steps for use cases involving private container images by using IBM Cloud Container Registry (ICR).
```

Comment:
Suggestions:

- Syntax/grammar changes (important)
- Change "This guide describe[s] steps for use cases" -> to maybe "This guide describes use cases..." or "This guide covers certain use cases ..."
- "by using IBM ... Registry": The image is stored

For instance:

```
This guide covers certain scenarios involving using private container images with the IBM Cloud Container Registry (ICR).
```

Note that you're using "Scenarios" late on in the guide, so I'm suggesting to reuse it here.

Since these are some of potentially many use cases or scenarios, I've added "certain".